java/pom.xml

```diff
@@ -35,7 +35,8 @@
     <dep.netty.version>4.1.22.Final</dep.netty.version>
     <dep.jackson.version>2.7.9</dep.jackson.version>
     <dep.hadoop.version>2.7.1</dep.hadoop.version>
-    <fbs.version>1.2.0-3f79e055</fbs.version>
+    <fbs.version>1.9.0</fbs.version>
+    <flatc.version>1.9.0</flatc.version>
```

Comment:
I guess we want to keep a separate version for flatc in case we want to upgrade flatbuffers but keep the same compiler?

Reply:
I think we should try best to keep them the same but two situations that they can be different:
* The one you mentioned
* If we need to update the flatc executables for the same flatbuffer version, we can release version 1.9.0-1 for example for flatc

Reply:
Yeah, sounds fine. While we are making changes, do you think we should add the `dep` prefix in the property, e.g. `<dep.fbs.version>` just to be consistent? Flight also defines the protobuf version as `<dep.protobuf.version>`

Reply:
Sounds good. Let me do that.